src/app/Library/CrudPanel/CrudButton.php

```diff
@@ -162,6 +162,32 @@ public function view($value)
         return $this->save();
     }
 
+    /**
+     * Set the button position. Defines where the button will be shown
+     * in regard to other buttons in the same stack.
+     *
+     * @param  string  $stack  'beginning' or 'end'
+     * @return CrudButton
+     */
+    public function position($position)
+    {
+        switch ($position) {
+            case 'beginning':
+                $this->makeFirst();
+                break;
+
+            case 'end':
+                $this->makeLast();
+                break;
+
+            default:
+                abort(500, "Unknown button position - please use 'beginning' or 'end'.");
+                break;
+        }
+
+        return $this->save();
```

Comment:
Actually we don't need to save anything, since that's done in the methods above:

```suggestion
        return $this;
```